source/install/cloud-changelog.md

```diff
@@ -4,12 +4,33 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-08-18](#release-2022-08-18)
 - [Release 2022-08-10](#release-2022-08-10)
 - [Release 2022-08-03](#release-2022-08-03)
 - [Release 2022-07-28](#release-2022-07-28)
 - [Release 2022-07-20](#release-2022-07-20)
 - [Release 2022-07-13](#release-2022-07-13)
-- [Release 2022-06-29](#release-2022-06-29)
+
+## Release 2022-08-18
+
+### Improvements
+
+#### User Interface (UI)
+ - Only the most recent message is now marked as unread when marking a thread as unread from the Threads list.
+ - Insights filters now persist instead of reset to default when switching to channels and returning back to Insights view.
```

Comment:
```suggestion
 - Insights filters now persist instead of being reset to default when switching to channels and returning back to the Insights view.
```